.github/workflows/checks.yml

```diff
@@ -28,3 +28,33 @@ jobs:
       - run: yarn
       - run: yarn run prettify:diff
       - run: yarn run lint
+  mobile-tests:
+    name: Mobile Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: true
+      - name: Install Yarn dependencies
+        run: yarn
+      - name: Build Yarn packages
+        run: yarn build --ignore docs
```

Comment:
Doesn't have to be addressed in this PR, but might be worth caching this, similar to what was done on CircleCI.

Reply:
yup, i plan to do the caching in a future PR. :)